airstack-modules/bin/index.ts

```diff
@@ -69,211 +69,18 @@ for (let index = START_INDEX; index < args.length; index++) {
 integrate(vertical, yaml, graphql, dataSources, templates)
   .then(() => {
     switch (vertical) {
-      case Vertical.Dex:
-        console.log(`
-        Integration done. Please call the following functions.
-        1. Add pool
-        
-          function addDexPool(
-            poolAddress: string,
-            fee: BigInt,
-            inputTokens: Array<string>,
-            weights: Array<BigDecimal> | null = null,
-            outputToken: string | null = null
-          ): void
-
-        2. Add pool
-
-          function swap(
-            poolAddress: string,
-            inputAmounts: Array<BigInt>,
-            outputAmounts: Array<BigInt>,
-            inputTokenIndex: i32,
-            outputTokenIndex: i32,
-            from: string,
-            to: string,
-            hash: string,
-            logIndex: BigInt,
-            timestamp: BigInt,
-            blockNumber: BigInt
-          ): void
-
-          For documentation and examples please check: https://github.com/Airstack-xyz/Subgraphs
-        `);
-        break;
 
       case Vertical.NftMarketplace:
         console.log(`
-        Integration done. Please call the following functions.
-        
-        1. Track NFT trade transaction
-        trackNFTSaleTransactions(
-          chainID: string,
-          txHash: string,
-          txIndex: BigInt,
-          NftSales: Sale[],
-          protocolType: string,
-          protocolActionType: string,
-          timestamp: BigInt,
-          blockHeight: BigInt,
-          blockHash: string
-        )
-
-        For documentation and examples please check: https://github.com/Airstack-xyz/Subgraphs
+Integration for DOMAIN_NAME vertical done. For documentation and examples please check below link:
+https://github.com/Airstack-xyz/Subgraphs/blob/ens-airstack-module-refactor/airstack-modules/modules/airstack/nft-marketplace/Readme.md
```

Comment:
The URL should be pointing to main branch.

Reply:
done